sdk/metric/correct_test.go

```diff
@@ -241,28 +242,33 @@ func TestSDKLabelsDeduplication(t *testing.T) {
 		sum, _ := rec.Aggregator().(aggregator.Sum).Sum()
 		require.Equal(t, sum, core.NewInt64Number(2))
 
-		kvs := export.IteratorToSlice(rec.Labels().Iter())
+		kvs := rec.Labels().ToSlice()
 		actual = append(actual, kvs)
 	}
 
 	require.ElementsMatch(t, allExpect, actual)
 }
 
+func newSet(kvs ...core.KeyValue) *label.Set {
```

Comment:
nit: might want to change to `newSetIter` and return a `label.Iterator` given all calls use the returned `Iter()`.

Reply:
Done.